mmv1/templates/terraform/examples/subnetwork_reserved_internal_range.tf.erb

```diff
@@ -0,0 +1,22 @@
+resource "google_compute_subnetwork" "subnetwork_with_reserved_internal_range" {
+  name                    = "<%= ctx[:vars]['subnetwork_name'] %>"
+  region                  = "us-central1"
+  network                 = google_compute_network.default.id
+  reserved_internal_range = google_network_connectivity_internal_range.reserved_range.id
```

Comment:
```suggestion
  reserved_internal_range = google_network_connectivity_internal_range.reserved.id
```

Reply:
Interesting, the API spec suggests specifying the URL, but using the ID works too. Maybe we can update the parameter description? What do you think?